src/NHibernate.Test/Tools/hbm2ddl/SchemaValidator/SchemaValidateTableWithSchemaFixture.cs

```diff
@@ -0,0 +1,102 @@
+﻿using System;
+using System.Data;
+using NHibernate.Dialect;
+using NHibernate.Util;
+using NUnit.Framework;
+
+namespace NHibernate.Test.Tools.hbm2ddl.SchemaValidator
+{
+	[TestFixture]
+	public class SchemaValidateTableWithSchemaFixture : TestCase
+	{
+		protected override string MappingsAssembly => "NHibernate.Test";
+
+		protected override string[] Mappings => new[] { "Tools.hbm2ddl.SchemaValidator.VersionWithSchema.hbm.xml" };
+
+		protected override bool AppliesTo(Dialect.Dialect dialect)
+		{
+			switch (Dialect)
+			{
+				case MsSql2000Dialect _:
+				case PostgreSQLDialect _:
+				case SQLiteDialect _:
```

Comment:
SQL Ce uses the same pattern as SQLiteDialect

Reply:
Indeed, done two years later in e79f47e. Now I have checked `Dialect.Qualify` overrides, no other database does that, although Firebird and MySql do not support schema either.

(Firebird may be fixed by overriding `Qualify` for ignoring catalog and schema; I think it is better than introducing the hack in it. MySql should ideally be fixed for ignoring schema; but this may be a breaking change for people "benefiting" of the current behavior handling schema like a catalog (provided catalog is not supplied at the same time). Anyway both cases would have to be handled in another PRs.)